app/views/users/index.html.erb

```diff
@@ -1,3 +1,16 @@
 <div class="container">
-  <%= render partial: "user", collection: @users %>
-</div>
\ No newline at end of file
+
+ <%= link_to "Add a new post", new_user_post_path(user_id: @user.id), class: "add-new-post"%>  
```

Comment:
- [ ] [OPTIONAL] 
Good job on making improvements! Although posts and comments can be created successfully for the first user, it appears some errors were introduced in an attempt to resolve issues from the previous review; The `Add a new post` is appearing at the show all users page and therefore it's impractical to select a specific user to add a post, unlike the screen indicated by the previous review. Let's ensure to add this button on the _users/show_  view instead.

**Add New Post button displaying on wrong page:**
![image](https://github.com/fickryiman/blog-app/assets/12745474/588a64ad-c6c4-4628-967c-7d0f7f24046f)